diff --git a/node_modules/@ledgerhq/hw-transport-webusb/lib-es/TransportWebUSB.js b/node_modules/@ledgerhq/hw-transport-webusb/lib-es/TransportWebUSB.js
index 830d2e2..26a9779 100644
--- a/node_modules/@ledgerhq/hw-transport-webusb/lib-es/TransportWebUSB.js
+++ b/node_modules/@ledgerhq/hw-transport-webusb/lib-es/TransportWebUSB.js
@@ -8,7 +8,7 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
     });
 };
 import Transport from "@ledgerhq/hw-transport";
-import hidFraming from "@ledgerhq/devices/hid-framing";
+import hidFraming from "@ledgerhq/devices/lib/hid-framing";
 import { identifyUSBProductId } from "@ledgerhq/devices";
 import { log } from "@ledgerhq/logs";
 import { TransportOpenUserCancelled, TransportInterfaceNotAvailable, TransportWebUSBGestureRequired, DisconnectedDeviceDuringOperation, DisconnectedDevice, } from "@ledgerhq/errors";
